app/Http/Controllers/Api/ExperimentController.php

```diff
@@ -59,32 +55,47 @@ public function create(Request $request)
 
         $id = Util::uuid();
         $name = $request->get('name');
-        $experiment_number = $request->get('experiment_number');
-        $experiment_intro = $request->get('experiment_intro');
-        $experiment_goal = $request->get('experiment_goal');
-        $experiment_mock = $request->get('experiment_mock');
-        $apparatus = $request->get('apparatus');
-        $experiment_resource = $request->get('experiment_resource');
-        $procedures = $request->get('procedures');
-        $exercise = $request->get('exercise');
-        $required = $request->get('required');
-        $theory = $request->get('theory');
-        $tables = $request->get('tables');
-        $graph = $request->get('graph');
+        //$experiment_number = $request->get('experiment_number');
+        $experiment_intro = $request->get('experiment_intro')??'-';
+        $experiment_goal = $request->get('experiment_goal')??'-';    
+        $experiment_diagram = $request->experiment_diagram;
+        return dd($experiment_diagram);
```

Comment:
@saidabdul80 why do we have a return statement here?

Reply:
forget to remove it. Noted!